app/src/main/res/values/styles.xml

```diff
@@ -141,4 +141,8 @@
     <item name="android:fontFamily">sans-serif-medium</item>
     <item name="textAllCaps">true</item>
   </style>
+
+  <style name="ToolbarTitle" parent="@style/TextAppearance.Widget.AppCompat.Toolbar.Title">
```

Comment:
There is already style present for toolbar text appearance `ToolbarTextAppearance`. Please if it is needed keep or remove it. Let's use common style everywhere.

Reply:
Thanks for picking this. Removed new style and used `ToolbarTextAppearance`